src/app/pages/log-in/log-in.component.scss

```diff
@@ -0,0 +1,18 @@
+@import "typography";
+@import "color";
+
+.forgot-password {
+  display: flex;
```

Comment:
it is good to use one approach for styling elements for all application(grid, flex, margin + position)

Reply:
@dmitry-darly-solutions 
Could you please explain what you mean?

Position + margin use when it requires design.
Flexbox create for single line design.
Grid create for matrix design.

Different approach use for different situation. If new approach didn't require, then we will still create site with help tables.

For example, how I should create input with eye icon inside input with help grid and without use position?

Reply:
@dmitry-darly-solutions Fixed.